webmails/roundcube/mailu.js

```diff
@@ -0,0 +1,9 @@
+rcmail.addEventListener('init', function(evt) {
+      var btn = document.createElement('a');
+      btn.setAttribute("href", "{{ WEB_ADMIN }}");
+      btn.setAttribute("role", "button");
+      btn.setAttribute("class", "showurl");
+      btn.innerHTML = '<span class="inner">To Mailu</span>';
```

Comment:
Please change this to
btn.innerHTML = '<span class="inner">To {{ SITENAME }}</span>';

Not everyone uses Mailu as the main site name. In SITENAME in mailu.env you can configure the name for the site. By default this is Mailu, but that is not the case for all users. Some users do not want Mailu branding at all.